thoth/messaging/__init__.py

```diff
@@ -37,12 +37,14 @@
 from .unresolved_package import UnresolvedPackageMessage
 from .unrevsolved_package import UnrevsolvedPackageMessage
 from .update_provides_src_distro import UpdateProvidesSourceDistroMessage
+from .kebechet_run_url import KebechetRunUrl
 
 ALL_MESSAGES = [
     AdviseJustificationMessage,
     AdviserReRunMessage,
     AdviserTriggerMessage,
     HashMismatchMessage,
+    KebechetRunUrl,
```

Comment:
all messages should end with Message, moreover it will be triggered as well, let's use `KebechetRunUrlTriggerMessage`